governance/polls/Atlas Edit Weekly Cycle Proposal - April 28, 2025.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Atlas Edit Weekly Cycle Proposal - April 28, 2025
+summary: This Atlas Edit adds documentation for a newly deployed Standby Spell that allows Sky Governance to disable the SP-BEAM in an emergency.
+discussion_link: https://forum.sky.money/t/atlas-edit-weekly-cycle-proposal-week-of-april-28-2025/26307
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 20000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2025-04-28T16:00:00
+end_date: 2025-05-01T16:00:00
+---
+# Atlas Edit Weekly Cycle Proposal - April 28, 2025
+
+The Governance Facilitators have placed an [Atlas Edit Weekly Cycle Proposal](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59|0db30308) into the [voting system](https://vote.makerdao.com/polling) [on behalf of Prime Delegate BLUE](https://forum.sky.money/t/atlas-edit-weekly-cycle-proposal-week-of-april-28-2025/26307/2). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59%7C0db30308) will be active for three days beginning on Monday, April 28 at 16:00 UTC.
```

Comment:
```suggestion
The Governance Facilitators have placed an [Atlas Edit Weekly Cycle Proposal](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59|0db30308) into the [voting system](https://vote.makerdao.com/polling) [on behalf of Prime Delegate BLUE](https://forum.sky.money/t/atlas-edit-weekly-cycle-proposal-week-of-april-28-2025/26307/2). This Governance Poll will be active for three days beginning on Monday, April 28 at 16:00 UTC.
```

We're linking to the same article twice here. It's confusing. We should probably edit the template.